doc/architecture/decisions/0004-reference-data-sync.md

```diff
@@ -245,80 +241,66 @@ These events are raised when changes are made to add or update incentive levels
 #### Get the details of an incentive level for a specified prison
 This contains spend limits and visit allowances
 
-`GET /incentive/levels/{level}/{prisonId}` -
+`GET incentive/prison-levels/{prisondId}/level/{level}` -
+e.g. **/incentive/prison-levels/MDI/level/STD** returns
 ```json
 {
-  "incentiveLevel": "EN4",
+  "levelCode": "STD",
+  "levelDescription": "string",
```

Comment:
Not a biggie at all: Maybe this was supposed to be `'Standard"` rather than `"string"`?

Reply:
Cf. https://github.com/ministryofjustice/hmpps-incentives-api/pull/387